accounts/keystore/keystore.go

```diff
@@ -374,7 +374,7 @@ func (ks *KeyStore) GenerateProofOfPossession(a accounts.Account) ([]byte, error
 	}
 	defer privateKey.Destroy()
 
-	signature, err := privateKey.SignPoP()
+	signature, err := privateKey.SignPoP(a.Address[:])
```

Comment:
Is this copy necessary? If so then could you add a comment for the benefit of anyone looking at this in the future (so they don't remove the copy).

Reply:
> Is this copy necessary? If so then could you add a comment for the benefit of anyone looking at this in the future (so they don't remove the copy).

You mean the `a.Address[:]` part? It's not a copy then, the `Address` is an array, and SignPoP expects a slice.

Reply:
Perhaps use `a.Address.Bytes()`?  Seems to be a bit more readable.